lib/build.rb

```diff
@@ -106,6 +106,7 @@ def query_pull_requests
       end
 
       aging_pull_requests_notification = true
```

Comment:
@lefticus I mimicked this initialization for my new variable, but I'm not sure if it was actually needed.